modules/installation-and-upgrade/pages/installation-and-upgrade-overview.adoc

```diff
@@ -0,0 +1,16 @@
+[[installation-and-upgrade-overview]]
+= Installation and Upgrade Guide Overview
+
+**Updated:** {docdate}
+
+This book provides guidance on installing and upgrading {productname} Server and Proxy.
+It is split into four sections:
+
+* Requirements: Describes the hardware, software, and networking requirements that you require before you begin.
+* Installation: Describes the process to install {productname} components.
+* Setting Up: Describes the initial steps you need to take after installation to make your {productname} environment ready to use.
```

Comment:
Rename to "Setup"

Reply:
@jcayouette - good catch, I can change it. However, "Setting Up" was actually inherited from previous versions.